docs/introduction/blockchain-representation.rst

```diff
@@ -2,51 +2,51 @@
 Internal representation of a Blockchain
 ***************************************
 
-Being a distributed architecture, the sidechain software is meant to be delivered as a software application that will be compiled/installed by potentially many different independent, connected computers. In blockchain jargon, these computers are called “Nodes,” and the term “node” is also generally used to name the blockchain software itself.
-So, the output of the Sidechain SDK, when customized by a developer, is a “Node” that implements core functionalities, and the added logic.
+Being a distributed architecture, the sidechain software is delivered as an application that will be compiled/installed on potentially many different independent, networked computers. In blockchain jargon, these computers are called “nodes,” and the term “node” is also generally used to name the blockchain software itself.
+So, the output of the Sidechain SDK, when customized by a developer, is a “node” that implements core functionalities and the added sidechain logic.
 
-A Node consists of 4 main elements: “**History**,” “**State**,” “**Wallet**,” and “**Memory pool**.” Before we get to know these 4 elements we need to know what a “box” is.
+A node consists of 4 main elements: “**History**,” “**State**,” “**Wallet**,” and “**Memory pool**.” Before we get to know these 4 elements we need to know what a “box” is.
```

Comment:
@zennermac 

A node consists of 4 main elements: “**history**,” “**state**,” “**wallet**,” and “**memory pool**.” Before we get to know these 4 elements we need to know what a “box” is.